app/src/main/java/swati4star/createpdf/fragment/PdfToImageFragment.java

```diff
@@ -19,9 +21,14 @@
 import android.widget.RelativeLayout;
 import android.widget.TextView;
 
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.RecyclerView;
+
 import com.afollestad.materialdialogs.MaterialDialog;
 import com.airbnb.lottie.LottieAnimationView;
 import com.dd.morphingbutton.MorphingButton;
+import com.google.android.material.bottomsheet.BottomSheetBehavior;
```

Comment:
[I repeat] Any beautification work should be done in separate commit may or may not in single pull request.

Reply:
+1